lib/helpers.js

```diff
@@ -129,8 +129,8 @@ async function getApkanalyzerForOs (sysHelpers) {
  * Checks mShowingLockscreen or mDreamingLockscreen in dumpsys output to determine
  * if lock screen is showing
  *
- * A note: `adb shell dumpsys trust` has better peformacne to detect if the screen
- * is locked with `adb dumpsys window` and grep it.
+ * A note: `adb shell dumpsys trust` performs better while detecting the locked screen state
+ * in comparison to detect if the screen is locked with `adb dumpsys window` and output parsing it.
```

Comment:
in comparison to `adb dumpsys window` output parsing.

Reply:
thanks so much